### PR TITLE
The Inactive Fella

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by KrisXV** [comment]
uwu

**Comment by ghost** [comment]
uWu

File: data/mods/ssb/abilities.js [hunk 1]
```diff
@@ -389,6 +389,17 @@ let BattleAbilities = {
...
 	},
+	// inactive
+	souleater: {
+		desc: "Attacking moves heal the user 33% of damage dealt.",
+		shortDesc: "Attacking moves heal the user 33% of damage dealt.",
+		id: "souleater",
+		name: "Soul Eater",
+		isNonstandard: "Custom",
+		onModifyMove(move) {
+			move.drain = [1, 3];
```
Comment (HoeenCoder) [inline data/mods/ssb/abilities.js:400]: Dose this interact well with status moves?

File: data/mods/ssb/moves.js [hunk 2]
```diff
@@ -1599,6 +1599,34 @@ let BattleMovedex = {
...
+		onTryMove() {
+			this.attrLastMove('[still]');
+		},
+		onPrepareHit(target, source) {
+			this.add('-anim', source, "Mean Look", target);
+		},
+		onHit(target, source, move) {
+			this.boost({atk: 2}, source, source);
+			target.trySetStatus('par', source);
+			target.addVolatile('trapped', source, move, 'trapper');
```
Comment (HoeenCoder) [inline data/mods/ssb/moves.js]:
```suggestion
			target.tryTrap();
```

**Comment by Torracat** [comment]
what’s the difference between this and Spirit Shackle’s code?

File: data/mods/ssb/moves.js (hunk 2 above)
Comment (ghost) [reply data/mods/ssb/moves.js]: what’s the difference between this and the code i pulled from Spirit Shackle? @HoeenCoder

File: data/mods/ssb/abilities.js (hunk 1 above)
Comment (ghost) [reply data/mods/ssb/abilities.js:400]: Yes, i tested it and it doesn’t do anything harmful or beneficial regarding status moves

File: data/mods/ssb/moves.js (hunk 2 above)
Comment (HoeenCoder) [reply data/mods/ssb/moves.js]: Was unaware this was pulled from spirit shackle.
This appears to be simpler though, not sure what the difference is.

Comment (ghost) [reply data/mods/ssb/moves.js]: ok fair enough

**Pull request merged**